web/src/lib/components/forms/edit-user-form.svelte

```diff
@@ -131,6 +132,17 @@
 			</p>
 		</div>
 
+		<div class="m-4 flex flex-col gap-2">
```

Comment:
We should add some description here of how the file path would be i.e Is it relative or absolute path? If it is relative, where can it be reference from?

Reply:
Adding a note here.  Let me know what you think of the  phrasing.